.github/workflows/cj-action-release.yml

```diff
@@ -31,6 +34,8 @@ jobs:
           # Create the Git tag
           git tag ${{ github.event.inputs.version }}
```

Comment:
Is it possible to also check existing versions, so that there is no clash?

Reply:
The workflow will fail if there is a clash.